robyn/openapi.py

```diff
@@ -0,0 +1,267 @@
+import json
+import os
+import re
+from inspect import Signature
+from typing import List, Dict, Union, Any, TypedDict
+
+import robyn
+
+
+class OpenAPI:
```

Comment:
and maybe we can initialise it as a dataclass. to make the interface look prettier.

Reply:
https://github.com/sparckles/Robyn/blob/6f76c64db2425ace6060a663bbd199e2f9373385/robyn/openapi_middleware.py

Have a look at this PR . This implementation looks much cleaner. We should include this in this PR

Reply:
we can use pydantic here instead of dataclass so we can slowly move to pydantic.